README.md

```diff
@@ -117,40 +216,106 @@ aws serverlessrepo put-application-policy \
 
 ## Install Lambda into AWS
 
+This lambda is intended to run in a stand-alone account. When using AWS
+Organizations, deploying the lambda to the payer account will aggregate costs
+from all member accounts. To get costs for a single member account, deploy the
+lambda to that member account.
```

Comment:
we now have two lambdas to generate cost reports. when you say `the lambda` do you mean this lambda? or the other lambda?

Reply:
yeah, when I say 'the lambda' I mean 'the lambda in this repo'